core/src/main/java/com/sequenceiq/cloudbreak/controller/v4/AuditEventV4Controller.java

```diff
@@ -30,6 +34,7 @@
     private CloudbreakRestRequestThreadLocalService threadLocalService;
 
     @Override
+    @CheckPermissionByAccount(action = AuthorizationResourceAction.DESCRIBE_ENVIRONMENT)
     public AuditEventV4Response getAuditEventById(Long workspaceId, Long auditId) {
```

Comment:
are audit events only generated for environments? if not, then this `AuthorizationResourceAction` possibly not a good choice for datalakes, datahubs, etc.

Reply:
for example, a DatahubUser possibly won't be able to query audit event of the related datahub